README.md

```diff
@@ -166,7 +170,8 @@ import Style from 'react-style-tag';
 Style.setGlobalOptions({
   doNotPrefix: true,
   hasSourceMap: true,
-  isMinified: true
+  isMinified: true,
+  autoPrefixerOpts: { remove: true }
```

Comment:
no abbreviations please ... `autoprefixerOptions` (also do not camelCase `autoprefixer`, as it does not align 1-to-1 with the package itself).